testcharms/charm.go

```diff
@@ -8,14 +8,19 @@ package testcharms
 import (
 	"archive/zip"
 	"bytes"
+	"fmt"
+	"github.com/juju/utils/fs"
```

Comment:
Bad grouping here... for next time.

Reply:
Sorry, this somehow got lost. Next time :+1: